cibuild.cmd

```diff
@@ -12,9 +12,11 @@ if /I "%1" == "/release" set BuildConfiguration=Release&&shift&& goto :ParseArgu
 call :Usage && exit /b 1
 :DoneParsing
 
-call "C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\Tools\VsDevCmd.bat"
+call "%VS140COMNTOOLS%VsDevCmd.bat"
 
 msbuild /v:m /m %AnalyzersRoot%\BuildAndTest.proj /p:CIBuild=true /p:Configuration=%BuildConfiguration%
+
+@echo off
```

Comment:
I thought it might affect the results of the build but now it wouldn't. Will Turn 
echo off for the whole file